spec/integration/post_spec.rb

```diff
@@ -2,7 +2,8 @@
 
 RSpec.describe 'Posts', type: :feature do
   before(:each) do
-    @user = User.create(name: 'John', photo: 'www.example.com', bio: 'Life science', posts_counter: 0)
+    @user = User.create(name: 'John', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Life science',
+                        posts_counter: 0)
```

Comment:
- [ ] The tests files need to be separated for each page, kindly update this
![image](https://github.com/shaaibu7/Blog-app/assets/62705947/a225ea8e-d8f2-4283-b614-28747993cb6b)

Reply:
Needs attention!